shared/src/testFixtures/groovy/gov/hhs/cdc/trustedintermediary/context/TestApplicationContext.groovy

```diff
@@ -19,8 +19,9 @@ class TestApplicationContext extends ApplicationContext {
         TEST_ENV_VARS.clear()
     }
 
-    def static injectRegisteredImplementations() {
-        injectRegisteredImplementations(true)
+    def static injectRegisteredImplementations(def skip = true) {
+        skipMissingImplementations = skip
```

Comment:
I like this.  But I also have a faint recollection that there's something wrong with this approach.  I'll see if I can't bang my head against the wall enough to remember what it was.